src/user/user.action.js

```diff
@@ -85,13 +85,36 @@ export const checkFollowStatus = url => {
   };
 };
 
+export const getFollowers = user => {
+  return (dispatch, getState) => {
+    const accessToken = getState().auth.accessToken;
+
+    dispatch({ type: GET_FOLLOWERS.PENDING });
+
+    fetchUrl(`${USER_ENDPOINT(user.login)}/followers?per_page=100`, accessToken)
+      .then(data => {
+        dispatch({
+          type: GET_FOLLOWERS.SUCCESS,
+          payload: data,
+        });
+      })
+      .catch(error => {
+        dispatch({
+          type: GET_FOLLOWERS.ERROR,
+          payload: error,
+        });
+      });
+  };
+};
+
 export const getUserInfo = user => {
   return dispatch => {
     return dispatch(getUser(user)).then(() => {
       dispatch(getOrgs(user));
       dispatch(
         checkFollowStatus(`https://api.github.com/user/following/${user}`)
       );
+      dispatch(getFollowers(user));
```

Comment:
I'm assuming we fire this to get the list of followers _as soon_ as a user navigates to the user profile screen so that we can determine whether we're following them correct? If that's the case, it's also currently being fired in the [`follower-list.screen`](https://github.com/gitpoint/git-point/blob/80495531c8307aabe3b486b53a2833bac9d68a96/src/user/screens/follower-list.screen.js#L29) so we'll need to remove it there to prevent it being called unnecessarily for the second time.